docs/configuration/integrations/integrations-next/app-o11y-exporter.md

```diff
@@ -55,15 +57,10 @@ Full reference of options:
     [host: <string> | default = "0.0.0.0"]
     [port: <number> | default = 8080]
 
-  # User defined prometheus metrics to be scraped. The sending end of the agent can
-  # update this metrics using the specified payload. Since these are performance metrics
-  # a Summary type metric is used internally for each metric defined
-  custom_measurements:
```

Comment:
Why is this removed?

Reply:
Any metrics should be defined by the frontend. AFAIK no other tool requires making backend agent config changes to add a metric.
 Also as discussed, removed prometheus exporter for now. Imho if we do a prometheus exporter later, measurements should be registered dynamically based on what front is sending

Reply:
Then we should remove the [respective config](https://github.com/grafana/agent/pull/1301/files#diff-4deef3ff88ba2ad90cc619164577b484819249b25747d2821763e264055427dfR48-R51) as well

Reply:
done!